.github/label-commenter-config.yml

```diff
@@ -0,0 +1,15 @@
+labels:
+  - name: needs-docs
+    labeled:
+      pr:
+        body: |
+          Thanks for your pull request, before this can be merged - corresponding documentation for your module is required:
+          - [Writing Module Documentation](https://github.com/rapid7/metasploit-framework/wiki/Writing-Module-Documentation)
+          - [Template](https://github.com/rapid7/metasploit-framework/blob/master/documentation/modules/module_doc_template.md)
+          - [Examples](https://github.com/rapid7/metasploit-framework/tree/master/documentation/modules)
+        action: open
```

Comment:
If you want to do nothing after commenting, you can remove this line `action: open`.

cf. https://github.com/peaceiris/actions-label-commenter/blob/547c566e1705f7f1df682811f10672345406c49f/src/main.ts#L130

Reply:
Sorry, I released v1.3.2 now. With this release, we can remove `action: open`.